### ISSUE TITLE
Some small issues about this build.

### ISSUE THREAD

**Issue by Thomas-ai-sudo** [Rsyd58/File_Rom#1]
Hello! I really like this ROM and I appreciate your time and effort you put to make it!
I just wanted to inform you about some bugs I found. First of all is the brightness. Whenever you set the brightness all the way down, the screen becomes completely black. Good thing that I had auto brightness enabled. Another thing is the RAM usage. I know this is a HyperOS build with Android 14 but is there a way to reduce RAM usage? I have nothing opened and it says 1Gb left. Last but not least, will you continue to update this ROM? Thanks!
Sincerely,
Thomas

**Comment by MIKAILREI** [comment]
for today, hyper mint is taked down for unknown reasons, I suggest you trying a different Rom, beside it, it's not very "secured".

**Comment by Thomas-ai-sudo** [comment]
Hey! Should I change ROM? Damn and it was a very well developed ROM and I really liked it. Let's hope they bring it back.

**Comment by MIKAILREI** [comment]
@Thomas-ai-sudo yeah likely you should, the Rom was take down because of drama so it will not likely going back up.

**Comment by MIKAILREI** [comment]
also the build is permissive, so dangerous.
